src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -3247,6 +3255,166 @@ closedir(DIR *dirp)
 	}
 }
 
+long
```

Comment:
[style] better use off64_t or off_t.

Reply:
Thank you! "long" is used to be consistent with the type defined in libc.

/usr/include/dirent.h
extern long int telldir (DIR *__dirp) __THROW __nonnull ((1));